### PR TITLE
[Cognitive Services] Add .nuspec file to include notice.txt

### PR DESCRIPTION
**Pull request by felixwa**
Add .nuspec file to include readme.txt

## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/CognitiveServices/Management.CognitiveServices/Microsoft.Azure.Management.CognitiveServices.nuspec [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        <dependency id="System.Xml.XDocument" version="[4.3.0, )" />
+        <dependency id="System.Runtime.Serialization.Primitives" version="[4.1.1, )" />
+        <dependency id="Microsoft.Rest.ClientRuntime.Azure" version="[3.3.5, 4.0.0)" />
+        <dependency id="Microsoft.Rest.ClientRuntime" version="[2.3.5, 3.0.0)" />
+      </group>
+    </dependencies>
+  </metadata>
+  <files>
+    <file src="bin\release\net452\Microsoft.Azure.Management.CognitiveServices.dll" target="lib\net452" />
+    <file src="bin\release\netstandard1.4\Microsoft.Azure.Management.CognitiveServices.dll" target="lib\netstandard1.4" />
```
Comment (felixwa) [inline src/SDKs/CognitiveServices/Management.CognitiveServices/Microsoft.Azure.Management.CognitiveServices.nuspec]: The "src" path works on my local machine but I'm not very sure if I should specify it in this way for "official" build. Please confirm.

File: src/SDKs/CognitiveServices/Management.CognitiveServices/Microsoft.Azure.Management.CognitiveServices.nuspec [hunk 2]
```diff
@@ -0,0 +1,62 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (felixwa) [inline src/SDKs/CognitiveServices/Management.CognitiveServices/Microsoft.Azure.Management.CognitiveServices.nuspec]: This .nuspec file is authored based on the file I extracted from Cognitive Services package in nuget.org

File: src/SDKs/CognitiveServices/Management.CognitiveServices/Microsoft.Azure.Management.CognitiveServices.nuspec [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    <description>Microsoft Azure Management Cognitive Services Library</description>
+    <summary>Microsoft Azure Management Cognitive Services Library.</summary>
+    <tags>Microsoft Azure Cognitive Services management,Cognitive Services,Cognitive Services management,REST HTTP client,windowsazureofficial,netcore451511</tags>
+    <dependencies>
+      <group targetFramework=".NETFramework4.5">
+        <dependency id="Microsoft.Rest.ClientRuntime.Azure" version="[3.3.5, 4.0.0)" />
+        <dependency id="Microsoft.Rest.ClientRuntime" version="[2.3.5, 3.0.0)" />
+      </group>
+      <group targetFramework=".NETStandard1.4">
+        <dependency id="Microsoft.NETCore.Platforms" version="[1.1.0, )" />
```
Comment (felixwa) [inline src/SDKs/CognitiveServices/Management.CognitiveServices/Microsoft.Azure.Management.CognitiveServices.nuspec]: Not sure if we really need this long dependency list?

File: src/SDKs/CognitiveServices/Management.CognitiveServices/Microsoft.Azure.Management.CognitiveServices.nuspec (hunk 1 above)
Comment (shahabhijeet) [inline src/SDKs/CognitiveServices/Management.CognitiveServices/Microsoft.Azure.Management.CognitiveServices.nuspec]: @felixwa were you able to create a nuget and install the new nuget and see the notice get's added to the users Solution in Visual Studio?

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

**Comment by shahabhijeet** [comment]
@azuresdkci add to whitelist

File: src/SDKs/CognitiveServices/Management.CognitiveServices/Microsoft.Azure.Management.CognitiveServices.nuspec (hunk 1 above)
Comment (shahabhijeet) [inline src/SDKs/CognitiveServices/Management.CognitiveServices/Microsoft.Azure.Management.CognitiveServices.nuspec]: @felixwa a good way to make this change is good is to create a signed nuget package using the signing job to create a signed package and test out in SDK and PowerShell scenarios.

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

**Comment by felixwa** [comment]
@shahabhijeet , I have removed that .nuspec file because I found it is not a must have to include the notice.txt file in the package. You can refer to this build artifacts: http://azuresdkci.cloudapp.net/view/NetCore-SDK/job/NetCore-SDK-Sign/156/

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**